app-backend/db/src/main/scala/com/azavea/rf/database/TemplateVersionDao.scala

```diff
@@ -0,0 +1,82 @@
+package com.azavea.rf.database
+
+import java.sql.Timestamp
+
+import com.azavea.rf.database.Implicits._
+import com.azavea.rf.datamodel._
+import doobie._
+import doobie.implicits._
+import doobie.postgres._
+import doobie.postgres.implicits._
+import cats._
+import cats.data._
+import cats.effect.IO
+import cats.implicits._
+import java.util.UUID
+
+import scala.concurrent.Future
+
+object TemplateVersionDao extends Dao[TemplateVersion] {
+  val tableName = "template_versions"
+
+  val selectF = sql"""
+    SELECT
+    FROM """ ++ tableF
+
+  def insert(
+    tvCreate: TemplateVersion.Create
+  ): ConnectionIO[TemplateVersion] = {
+    sql"""
+      INSERT INTO ${tableName}
```

Comment:
`${tableName}` can't be interpolated -- you'll need to separate that out into its own fragment